Server/ReasnAPI/ReasnAPI/Services/EventService.cs

```diff
@@ -218,7 +218,7 @@ public IEnumerable<ParticipantDto> GetEventParticipantsBySlug(string slug)
 
     public IEnumerable<CommentDto> GetEventCommentsBySlug(string slug)
     {
-        var eventToReturn = context.Events.Include(e => e.Tags).Include(e => e.Parameters).Include(e => e.Comments)
+        var eventToReturn = context.Events.Include(e => e.Comments)
             .Include(e => e.Participants).FirstOrDefault(e => e.Slug == slug);

```

Comment:
Do we neeed to include participants here?

Reply:
changed